### PR TITLE
Remove campaign analytics graphs.

### PR DESCRIPTION
**Pull request by LATaylor-guardian**
I added logging to check that nobody was using these which confirmed our assumptions. They also are expensive to load and don't add any known value.

### CODE REVIEW

**Review by kelvin-chappell — APPROVED** [review]
Looks good.  The job that generates the data behind the graphs will need to be removed as well, and perhaps the tables.

File: public/components/CampaignAnalytics/CampaignAnalytics.js [hunk 1]
```diff
@@ -1,97 +0,0 @@
...
-
-    if (!this.props.campaignPageViews) {
-      return <div className="campaign-info__body">Loading... </div>;
-    }
-
-    return (
-      <div>
-        <div className="campaign-info__body">
-
-          <CampaignPerformanceSummary campaign={this.props.campaign}
```
Comment (janua) [inline public/components/CampaignAnalytics/CampaignAnalytics.js:47]: Do we not want to keep CampaignPerformance? Just change the graph? Or maybe your plan is to completely re-write the graph?

**Comment by LATaylor-guardian** [comment]
> Looks good. The job that generates the data behind the graphs will need to be removed as well, and perhaps the tables.

Yes, I'll do that at a later date when we no for sure no one cares about these.

File: public/components/CampaignAnalytics/CampaignAnalytics.js (hunk 1 above)
Comment (LATaylor-guardian) [reply public/components/CampaignAnalytics/CampaignAnalytics.js:47]: Just planned to recreate (using other graphs as a reference)

**Pull request merged**